src/test/java/pro/sky/java/course2/homework04/constants/EmployeeServiceConstants.java

```diff
@@ -0,0 +1,15 @@
+package pro.sky.java.course2.homework04.constants;
+
+public class EmployeeServiceConstants {
+
+    public static final String FIRST_NAME1 = "Vasya";
+    public static final String LAST_NAME1 = "Pupkin";
+    public static final int DEP_NUMBER1 = 1;
+    public static final Double SALARY1 = 100_000d;
+    public static final String FIRST_NAME2 = "Petya";
```

Comment:
```suggestion
    public static final Double SALARY1 = 100_000d;
    
    public static final String FIRST_NAME2 = "Petya";
```
лучше разделять константы на группы